src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -81,6 +82,19 @@ public UniquePersonList(UniquePersonList source) {
         return Collections.unmodifiableList(internalList);
     }
 
+    /*
+     * Sort the list based on their name.
+     * "DEF,ghh,ZXC,abc" will become "abc,DEF,ghh,ZXC"
+     */
+
+    public void sort(){
+        Comparator<Person> personComparator = (personA, personB) -> {
+            Name personAName = personA.getName();
+            Name personBName = personB.getName();
+            return personAName.compareTo(personBName);
+        };
+        Collections.sort(internalList,personComparator);
```

Comment:
> Commas should be followed by a white space.

Refer to [CS2113T Java Coding Standard](https://nuscs2113-ay1819s2.github.io/website/coding-standards/java/index.html) for more information.